index.js

```diff
@@ -142,8 +142,16 @@ module.exports = {
   },
 
   buildGoogleMapsUrl(config) {
-    if (!(config && config.key)) {
-      this.warn('You must provide at least a Google Maps API key to ember-google-maps. Learn more: https://ember-google-maps.sandydoo.me/docs/getting-started');
+    if (!(config && config.key) || !(config && config.client)) {
```

Comment:
I think this line will trip the warning in the correct case where either a key or client id is used.

Also, if we set `config = {}`, then we can avoid checking for `config` each time. The url should still get built and Google Maps will also generate console warnings for us for missing keys.

On a separate note, would `clientId` be a better name than `client`?

Reply:
I fixed the first two comments. I think `client` is fine. Thats what the parameter is called on the google api script tag, so I think keeping it consistent is best.